Exercise02-Basics/Ivo-Nakov/BASICS02/BASICS02-001.js

```diff
@@ -0,0 +1,18 @@
+/*
+BASICS02-001
+Declare five variables.
+Assign them with the ASCII codes of the first five English capital alphabet characters.
+Print them on the console.
+*/
+
+let asciiA = 65;
+let asciiB = 66;
+let asciiC = 67;
+let asciiD = 68;
+let asciiE = 69;
+
+console.log(asciiA);
+console.log(asciiB);
+console.log(asciiC);
+console.log(asciiD);
+console.log(asciiE);
```

Comment:
That would print just the values of ascii{A..E} variables. The scope is to print the character values of these integers, e.g.

```js
console.log(String.fromCharCode(asciiA));    //  "A"
```